ProjectName/src/navigation/NavTabs.tsx

```diff
@@ -94,6 +94,12 @@ const Tabs = () => {
               color={focused ? 'purple' : 'gray'}
             />
           ),
+          headerStyle: {
+            backgroundColor: darkmode == 'dark' ? '#212529' : 'transparent',
```

Comment:
I would suggest avoid boolean variables as way to pick something if possible.
They are very hard to extend. Imagine we introduce a different type of theme, like `daily`. It would be really hard to update every occurence where we use such approach.
You could check our `color/theme` implementation and consider it as a reference.